other/Perl Jam 2 - RCE.bcheck

```diff
@@ -0,0 +1,35 @@
+metadata:
+    language: v2-beta
+    name: "Perl JAM 2"
+    description: "Perl JAM 2 - RCE"
+    author: "Andrej Šimko @ Accenture"
+    tags: "Perl JAM 2","Perl","Remote Code Execution","RCE"
+
+define: 
+    request_all_headers = `{base.request.headers}`
+    multi_line = `{regex_replace({request_all_headers},"[\s\S]+boundary=","")}`
+    boundary = `{regex_replace({multi_line},"[\r\n].*","")}`
+    request_original_body = `{base.request.body}`
+
+    #this should potentially be changed to work on Windows deployment
+    rce_payload = "/bin/bash%20-c%20ifconfig%20|"
+
+given request then
+    if "multipart/form-data" in {base.request.headers} and {base.request.method} matches "POST" then
```

Comment:
Really minor, but you could flip the order of these checks as the method check is cheaper.